examples/platforms/simulation/radio.c

```diff
@@ -1016,6 +1047,56 @@ otError otPlatRadioGetCoexMetrics(otInstance *aInstance, otRadioCoexMetrics *aCo
 }
 #endif
 
+uint64_t otPlatRadioGetNow(otInstance *aInstance)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
+    return otPlatTimeGet();
+}
+
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+static otError updateIeData(otInstance *aInstance)
+
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
+    otError error  = OT_ERROR_NONE;
+    uint8_t offset = 0;
+
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
```

Comment:
Duplicate with line 1057. Remove for now. We can add it back in link metrics PR.